src/topic.js

```diff
@@ -0,0 +1,29 @@
+const SNS = require('aws-sdk/clients/sns');
+const AWS = require('aws-sdk/global');
+
+class Topic {
+  constructor(arn, region) {    
+    this.arn = arn;
+    this.sns = new AWS.SNS({ region });
+  }
+
+  publish (data) {
+    let params = this.generateSnsMessage(data);
+    // console.log(params)
```

Comment:
Do we require a `console.log` here? If this is just for debug would be worth removing.

Reply:
This is fixed in https://github.com/lulibrary/LAG-Utils/pull/2/commits/502d786450479aefef2453f7086aa3459088afd1